pkg/controller/pxc/backup.go

```diff
@@ -34,7 +34,7 @@ func (r *ReconcilePerconaXtraDBCluster) reconcileBackups(ctx context.Context, cr
 	log := logf.FromContext(ctx)
 
 	backups := make(map[string]api.PXCScheduledBackupSchedule)
-	backupNamePrefix := backupJobClusterPrefix(cr.Name)
+	backupNamePrefix := cr.Namespace + "/" + backupJobClusterPrefix(cr.Name)
```

Comment:
I think it will not work `not exceed 22 characters`. Prefix can be too long.

Reply:
And this one:
`2023-12-11T18:45:28.109Z	ERROR	failed to create backup	{"controller": "pxc-controller", "namespace": "pxc1", "name": "cluster111111111222222", "reconcileID": "cb1268c3-34e5-4c88-a8f0-be64f93196f4", "error": "PerconaXtraDBClusterBackup.pxc.percona.com \"cron-cluster111111111-fs-pvc-20231211184528-1tfsi\" is invalid: metadata.labels: Invalid value: \"pxc1/ca65c-daily-backup\": a valid label must be an empty string or consist of alphanumeric characters, '-', '_' or '.', and must start and end with an alphanumeric character (e.g. 'MyValue',  or 'my_value',  or '12345', regex used for validation is '(([A-Za-z0-9][-A-Za-z0-9_.]*)?[A-Za-z0-9])?')"}`